app/repositories/conversation_repository.js

```diff
@@ -76,4 +77,22 @@ export default class ConversationRepository extends BaseRepository {
 
     return list;
   }
+
+  async showConversation(ws, requestId, conversation, participants, message) {
```

Comment:
I think we have to re-do it, cause `showConversation` name is a bit confusing.

Let's come up with a better name, e.g. `notifyAboutConversationCreate` and `notifyAboutConversationUpdate`

Reply:
Also, according to our architecture, we have the following relations:

`repository <-> controller <-> packetProcessor`.

So I think it's a controller's responsibility to talk to `packetProcessor`, not repository